.github/actions/environment/action.yml

```diff
@@ -8,6 +8,11 @@ runs:
         echo TEST_TIMEOUT=310 | tee -a $GITHUB_ENV
         echo NO_OUTPUT_TIMEOUT=320 | tee -a $GITHUB_ENV
         echo PRESERVE_ENVVARS=REPLICATION_SYNC_TIMEOUT,TEST_TIMEOUT,NO_OUTPUT_TIMEOUT | tee -a $GITHUB_ENV
+        # For self-hosted runners run w/o restart may need to kill
+        # hanged processes that could be leaved from the previous
+        # workflows runs.
+        ps auxT | grep " tarantool " | awk '{system("kill -9 "$2)}' || true
```

Comment:
`" tarantool "` will not match `/path/to/tarantool foo.lua`. Is it intentional?

Reply:
Right, it is intentional, in testing used only tarantool w/o path, so to be sure that script kills only needed processes let's use spaces around.

Reply:
Why then we need a space after tarantool?

Reply:
Just to be sure that found process exactly that was needed.